.gitattributes

```diff
@@ -0,0 +1,81 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto
+
+# Files with these extensions are accepted source files inside the ReactOS
+# tree. They should have native line endings of the OS.
+*.acf text
+*.asm text
+*.c text
+*.cc text
+*.cmake text
+*.cpp text
+*.cxx text
+*.h text
+*.hpp text
+*.hxx text
+*.idl text
+*.inc text
+*.inl text
+*.klc text
+*.l text
+*.lds text
+*.md text
+*.rc text
+*.s text
+*.S text
+*.sfd text
+*.spec text
+*.svg text
+*.txt text
+*.TXT text
+*.y text
+
+# Files with these extensions end up in the built ReactOS system, so they
+# need to have CRLF line endings.
+*.bat text eol=crlf
+*.cmd text eol=crlf
+*.exp text eol=crlf
+*.hhc text eol=crlf
+*.hhp text eol=crlf
+*.ini text eol=crlf
+*.INI text eol=crlf
```

Comment:
Is Git case-sensitive in this regard? (*.ini vs. *.INI)

Reply:
Git is whatever the OS is -- so in linux, it is case sensitive, and in windows, it's case-preserving.